elements/pfe-modal/src/pfe-modal.scss

```diff
@@ -0,0 +1,132 @@
+@import "../../pfe-sass/pfe-sass";
+$LOCAL: modal;
+
+$pfe-modal--breakpoint--medium: 640px;
+
+:host {
+	--pfe-modal--MaxWidth: 70vw;
+	--pfe-modal--MinWidth: 50vw;
+	--pfe-modal--MaxWidth--mobile: 94vw;
+	--pfe-modal--MaxHeight: 90vh;
+	--pfe-modal--Padding: calc(#{pfe-var(container-padding)} * 2) calc(#{pfe-var(container-padding)} * 3.5) calc(#{pfe-var(container-padding)} * 2) calc(#{pfe-var(container-padding)} * 2);
+	--pfe-modal__overlay--BackgroundColor: #{pfe-color(overlay)};
+	--pfe-modal__close--size: calc(#{pfe-var(ui--element--size)} - 4px);
+	display: block;
+	position: relative;
+
+   @media screen and (max-height: $pfe-modal--breakpoint--medium) {
+      --pfe-modal--Padding: 0 0 0 calc(#{pfe-var(container-padding)} * 2);
+   }
+}
+
+.pfe-modal {
+	&__outer {
+		display: flex;
+		position: fixed;
+		height: 100%;
+		width: 100%;
+		top: 0;
+		left: 0;
+		flex-direction: column;
+		align-items: center;
+		justify-content: center;
+		z-index: #{pfe-var($cssvar: modal, $map: $z-index)};
+
+		&[hidden] {
+			display: none;
+		}
+	}
+	&__overlay {
+		position: fixed;
+		height: 100%;
+		width: 100%;
+		top: 0;
+		left: 0;
+		background-color: pfe-local($cssvar: BackgroundColor, $region: overlay);
+		cursor: pointer;
+
+		&[hidden] {
+			display: none;
+		}
+	}
+	&__window {
+		position: relative;
+		max-width: pfe-local(MaxWidth--mobile);
+		min-width: pfe-local(MinWidth);
+		max-height: calc(100% + -32px);
```

Comment:
why this calculation instead of `var(--pfe-modal--MaxHeight)`?

Reply:
Ahh, good point. Fixed this.